### PR TITLE
fix(avatar): change avatar seed algorithm to sync with android

### PR DESCRIPTION
**Pull request by braddialpad**
Vue 3 https://github.com/dialpad/dialtone-vue/pull/1182

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,8 +47,6 @@
     "date-fns": "^2.30.0",
     "emoji-regex": "^10.2.1",
     "emoji-toolkit": "^6.6.0",
-    "fnv-plus": "^1.3.1",
-    "tippy": "^0.0.0",
```
Comment (braddialpad) [inline package.json:51]: Anyone know what this tippy version 0.0.0 is? very weird.

Comment (iropolo) [reply package.json:51]: @sanjeevdialpad https://github.com/dialpad/dialtone-vue/pull/1136 is it intended?

**Review by iropolo — APPROVED** [review]
Thanks, wait for @sanjeevdialpad clarification before merge.

**Comment by braddialpad** [comment]
Confirmed with Sanjeev that this can be removed.

**Pull request merged**